about.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to the About Page</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+  <header>
+    <h3 class="logo-text">My Logo</h3>
+    <nav>
+      <ul>
+        <li><a href="./portfolio.html">Portfolio</a></li>
+        <li><a href="./about.html">About</a></li>
+        <li><a href="./contact.html">Contact</a></li>
+      </ul>
+    <i class="fa-sharp fa-solid fa-bars"></i>
+    </nav>
+  </header>
```

Comment:
- [ ] As mentioned by the first reviewer please make sure your hamburger icon and logo are aligned. I suggest using flex on the header and then align logo and hamburger using  
```
       align-content: center; 
       align-item: center;
``` 
- [ ] Please also make sure that you use the correct font-size for your logo as mentioned by the previous reviewer
Your design should look as close to the Figma design as possible as mentioned in the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

>  Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)



**_Here is the Screenshot of your design:_**
<img width="375" alt="Screenshot 2023-02-16 at 9 58 38 AM" src="https://user-images.githubusercontent.com/44798044/219274976-a507336b-a7fc-4113-8d31-e0bf8e11107c.png">

_**and Figma Design:**_

![figma](https://user-images.githubusercontent.com/44798044/219274994-a7ac5b64-c84b-458d-883b-71458da7770b.jpeg)

Reply:
On the figma design, the logo size is 18px. I used 18px also but it seems you want me to still change from that 18px?

Reply:
I am still working on the alignment but kindly comment on the size issue for the logo

Reply:
Hi @shella12 , I am not sure you would be notified, that is why i used this @ ing method to reach out again